examples/pingpong/main.go

```diff
@@ -43,7 +43,7 @@ func main() {
 	// Wait here until CTRL-C or other term signal is received.
 	fmt.Println("Bot is now running.  Press CTRL-C to exit.")
 	sc := make(chan os.Signal, 1)
-	signal.Notify(sc, syscall.SIGINT, syscall.SIGTERM, os.Interrupt, os.Kill)
+	signal.Notify(sc, syscall.SIGTERM, os.Interrupt, os.Kill)
```

Comment:
Since `syscall.SIGTERM` is being left in, would it be more consistent to instead change `os.Interrupt` to `syscall.SIGINT` and `os.Kill` to `syscall.SIGKILL`?

```suggestion
	signal.Notify(sc, syscall.SIGTERM, syscall.SIGINT, syscall.SIGKILL)
```